tests/benchmarks/test_coiled.py

```diff
@@ -1,8 +1,14 @@
 import uuid
 
+import pytest
 from coiled import Cluster
 
 
+@pytest.fixture(autouse=True)
+def parquet_benchmark_fixture(benchmark_time):
```

Comment:
Why do we need this here? not sure I follow

Reply:
It automatically applies the `benchmark_time` fixture to all the tests in the module. It would be equivalent to just include it in the test itself. I suppose I was just looking ahead to when this module has multiple tests in it.